frontend/simple/views/components/Chatroom/ChatMain.vue

```diff
@@ -0,0 +1,328 @@
+<template>
+  <div class="c-chatmain" :class="{ isActive: summary.title }">
+    <template v-if="summary.title">
+      <chat-header
+        :title="summary.title"
+        :description="summary.description"
+        routerBack="/messages"
+      >
+        <template slot="actions">
+          <button class="button is-icon">
+            <i class="fa fa-user-plus"></i>
+          </button>
+          <button class="button is-icon">
+            <i class="fa fa-bell"></i>
+          </button>
+          <menu-parent class="level-right">
+            <menu-trigger class="is-icon">
+              <i class="fa fa-ellipsis-v"></i>
+            </menu-trigger>
+
+            <!-- TODO later - be a drawer on mobile -->
+            <menu-content class="c-actions-content">
+              <list hasMargin>
+                <menu-item tag="button" itemId="hash-1" icon="heart">
+                  Option 1
+                </menu-item>
+                <menu-item tag="button" itemId="hash-2" icon="heart">
+                  Option 2
+                </menu-item>
+                <menu-item tag="button" itemId="hash-3" icon="heart">
+                  Option 3
```

Comment:
Don't forget `<i18n>`s everywhere we have human-readable text :)

Reply:
Where I use dummy text (like this example) I didn't bother to use i18n because it will be changed in the future. Does it make sense?

Reply:
Eh, I still think it's important to add it here just so that we don't forget to add it. :)

Reply:
Okidoki, I'll add it then :)